README.md

```diff
@@ -1,24 +1,36 @@
-# Duke project template
-
-This is a project template for a greenfield Java project. It's named after the Java mascot _Duke_. Given below are instructions on how to use it.
-
-## Setting up in Intellij
-
-Prerequisites: JDK 11, update Intellij to the most recent version.
-
-1. Open Intellij (if you are not in the welcome screen, click `File` > `Close Project` to close the existing project first)
-1. Open the project into Intellij as follows:
-   1. Click `Open`.
-   1. Select the project directory, and click `OK`.
-   1. If there are any further prompts, accept the defaults.
-1. Configure the project to use **JDK 11** (not other versions) as explained in [here](https://www.jetbrains.com/help/idea/sdk.html#set-up-jdk).<br>
-   In the same dialog, set the **Project language level** field to the `SDK default` option.
-3. After that, locate the `src/main/java/Duke.java` file, right-click it, and choose `Run Duke.main()` (if the code editor is showing compile errors, try restarting the IDE). If the setup is correct, you should see something like the below as the output:
-   ```
-   Hello from
-    ____        _        
-   |  _ \ _   _| | _____ 
-   | | | | | | | |/ / _ \
-   | |_| | |_| |   <  __/
-   |____/ \__,_|_|\_\___|
-   ```
+# Anya Forger - Your Task Management Chat bot
+
+Anya Forger is a chat bot inspired by the character from the [_Spy x Family_](https://en.wikipedia.org/wiki/Spy_%C3%97_Family) anime. She is your ultimate task management companion. Whether you're a secret agent or just someone with a busy schedule, Anya can help you stay organized and on top of your tasks.
```

Comment:
![Amazed_Anya](https://github.com/nus-cs2103-AY2324S1/ip/assets/110957690/08b4c455-591f-48db-ae6e-cd57d2e99e56)
I like how you created your own design instead of Duke! I am a big fan of the anime as well!